zero/benches/intra-block-tries.rs

```diff
@@ -34,7 +32,7 @@ fn criterion_benchmark(c: &mut Criterion) {
                      block_trace,
                      other_data,
                  }| {
-                    trace_decoder::entrypoint(
+                    zero::intra_block_tries::entrypoint(
```

Comment:
Repeating the point from some previous PR that `entrypoint` is not a good API function name.